UMN-AutoPackager/private/Get-RedirectedUri.ps1

```diff
@@ -0,0 +1,48 @@
+function Get-RedirectedUri {
+    <#
+	.SYNOPSIS
+		Gets the real download URL from the redirection.
+	.DESCRIPTION
+		Used to get the real URL for downloading a file, this will not work if downloading the file directly.
+	.EXAMPLE
+		Get-RedirectedURI -URI "https://download.mozilla.org/?product=firefox-latest&os=win&lang=en-US"
+	.PARAMETER URL
+		URL for the redirected URL to be un-obfuscated
+	.NOTES
+		Code from: Redone per issue #2896 in core https://github.com/PowerShell/PowerShell/issues/2896
+	#>
```

Comment:
We should standardize on help for functions being outside right above the function.